Source/WebCore/platform/cocoa/PlaybackSessionModel.h

```diff
@@ -28,6 +28,7 @@
 #if PLATFORM(IOS_FAMILY) || (PLATFORM(MAC) && ENABLE(VIDEO_PRESENTATION_MODE))
 
 #include "PlatformMediaSession.h"
+#include "VideoReceiverEndpoint.h"
```

Comment:
This should be forward-declarable.

Reply:
This header just contains a `using` declaration.